src/Features/Core/Portable/NavigateTo/NavigateToSearcher.cs

```diff
@@ -70,13 +71,15 @@ internal async Task SearchAsync()
                 // order.  That way we provide results for the documents the user is working
                 // on faster than the rest of the solution.
                 var docTrackingService = workspace.Services.GetService<IDocumentTrackingService>();
+
+                var seenItems = new HashSet<INavigateToSearchResult>(NavigateToSearchResultComparer.Instance);
```

Comment:
It seems the declaration of the hashset can be moved into `SearchProjectsInPriorityOrderAsync` and `SearchAllProjectsAsync`, so it's closer to the usage and simplify the method signatures.

Reply:
indeed.  i'll do that.

Reply:
done.  also unified the so SearchProjects methods.